plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -662,8 +687,8 @@ public static Map<String, String> createTableProperties(ConnectorTableMetadata t
         if (!columns.isEmpty()) {
             checkFormatForProperty(fileFormat.toIceberg(), FileFormat.ORC, ORC_BLOOM_FILTER_COLUMNS);
             validateOrcBloomFilterColumns(tableMetadata, columns);
-            propertiesBuilder.put(ORC_BLOOM_FILTER_COLUMNS_KEY, Joiner.on(",").join(columns));
-            propertiesBuilder.put(ORC_BLOOM_FILTER_FPP_KEY, String.valueOf(getOrcBloomFilterFpp(tableMetadata.getProperties())));
+            propertiesBuilder.put(TableProperties.ORC_BLOOM_FILTER_COLUMNS, Joiner.on(",").join(columns));
```

Comment:
static import

Reply:
```
io.trino.plugin.iceberg.IcebergTableProperties.ORC_BLOOM_FILTER_COLUMNS
org.apache.iceberg.TableProperties.ORC_BLOOM_FILTER_COLUMNS
```
We have a name conflict here so we can't static import both.

I thought about refactor the mapping logic between Trino session properties and Iceberg table properties.
The current mapping logic is a bit messy and across multiple classes.
However, the changes will be big so it's better to do it in another PR.

Reply:
@oneonestar What do you think about suffixing the ORC Trino session properties with _PROPERTY (as is already done with the other properties) and importing ORC_BLOOM_FILTER_COLUMNS and ORC_BLOOM_FILTER_FPP from org.apache.iceberg.TableProperties into IcebergTableProperties, and then importing them here?

This approach would maintain the pattern that already exists, and it would make things easier for me in the #20393 PR to add the constant that holds the iceberg table properties.

Subsequently, we could revisit how we deal with both kinds of properties.

What are your thoughts on this?

Reply:
@ndrluis Thanks for the suggestion.
Rename Trino session properties with `_PROPERTY` make a lot of sense.

> importing ORC_BLOOM_FILTER_COLUMNS and ORC_BLOOM_FILTER_FPP from org.apache.iceberg.TableProperties into IcebergTableProperties, and then importing them here?

I don't understand why these properties are needed in IcebergTableProperties.

I did the change in https://github.com/trinodb/trino/pull/20433/commits/eddb7e8ab7e4052ea378f099553ca01f7548e3ff . Please take a look.